bundles/org.openhab.core.boot/bnd.bnd

```diff
@@ -1,2 +1 @@
 Bundle-SymbolicName: ${project.artifactId}
```

Comment:
This file only existed for adding the activator to the manifest. So it should be removed now.